_build_jdk_image.sh

```diff
@@ -28,8 +28,8 @@ function _build_docker_image {
 	DOCKER_IMAGE_TAGS+=("${LIFERAY_DOCKER_REPOSITORY}/${jdk_image_name}:${image_version}-${TIMESTAMP}")
 	DOCKER_IMAGE_TAGS+=("${LIFERAY_DOCKER_REPOSITORY}/${jdk_image_name}")
 
-	local zulu_amd64_version="LIFERAY_DOCKER_ZULU_${jdk_version}_AMD64_VERSION"
-	local zulu_arm64_version="LIFERAY_DOCKER_ZULU_${jdk_version}_ARM64_VERSION"
+	local zulu_amd64_version="LIFERAY_DOCKER_ZULU_AMD64_VERSION"
+	local zulu_arm64_version="LIFERAY_DOCKER_ZULU_ARM64_VERSION"
```

Comment:
I removed the number so we can use the variable from the shared build_jdk_image function. When declaring we can't use names like LIFERAY_DOCKER_ZULU_${jdk_version}_ARM64_VERSION, a variable in a variable name, so I found it easier to just use a shared public variable.

Reply:
Yes, you can, @Wykenz! Can you try this `LIFERAY_DOCKER_ZULU_${!jdk_version}_ARM64_VERSION`?

Thanks